src/components/NavBar/NavBar.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <header id="site-header" class="navbar-container">
+    <nav class="flex flex-row align-items-center justify-content-between w-full">
+      <div id="navBarRightEnd" class="flex flex-row align-items-center justify-content-start w-full gap-1">
+        <div class="flex align-items-center justify-content-center w-full">
+          <PvMenubar :model="menuItems" class="w-full">
+            <template #start>
+              <router-link :to="{ path: APP_ROUTES.HOME }" data-cy="navbar__logo">
+                <div class="navbar-logo mx-3">
+                  <ROARLogo v-if="!logo" />
+                  <PvImage v-else :src="logo" alt="Logo" width="200" height="auto" />
+                </div>
+              </router-link>
+            </template>
+
+            <template #menubuttonicon>
+              <PvButton
+                icon="pi pi-bars mr-2"
+                class="bg-primary text-white p-2 mr-2 border-none border-round hover:bg-red-900"
+                label="Menu"
+                @click="toggleMenu"
+              />
+            </template>
+
+            <template #end>
+              <div class="flex gap-2 align-items-center justify-content-center mr-3">
+                <div v-if="isWideScreen" class="nav-user-wrapper flex align-items-center gap-2 bg-gray-100">
+                  <div class="text-lg font-bold text-gray-600" data-cy="navbar__display-name">
+                    {{ $t('navBar.greeting') }}, {{ displayName }}!
+                  </div>
+
+                  <PvButton
+                    text
+                    data-cy="navbar__signout-btn-desktop"
+                    class="no-underline h-2 p-1 m-0 text-primary border-none border-round h-2rem text-sm hover:bg-red-900 hover:text-white"
+                    @click="handleSignOut"
+                    >{{ $t('navBar.signOut') }}
+                  </PvButton>
+                </div>
+
+                <PvButton
+                  v-else
+                  data-cy="navbar__signout-btn-mobile"
+                  class="no-underline m-0 bg-primary text-white border-none border-round h-2rem text-sm hover:bg-red-900"
+                  @click="handleSignOut"
+                >
+                  {{ $t('navBar.signOut') }}
+                </PvButton>
+
+                <div v-if="showAccountSettingsLink" class="nav-user-wrapper bg-gray-100">
+                  <router-link :to="{ path: APP_ROUTES.ACCOUNT_PROFILE }" data-cy="navbar__account-settings-btn">
```

Comment:
```suggestion
                  <router-link :to="{ path: APP_ROUTES.ACCOUNT_PROFILE }" data-cy="navbar__btn-account-settings">
```
Even though `data-cy` isn't really a CSS class, my brain seems to processes the block-element-modifier convention more naturally than other conventions. Here the block is the navbar, and the element is the button whose function is related to account settings. 

Ultimately this is a preference and how I have tried to style the data-cy attribute in the past, so I still approve of the changes regardless.